tests/Yaapii.Atoms.Tests/IO/TeeInputTest.cs

```diff
@@ -31,6 +31,61 @@ namespace Yaapii.Atoms.IO.Tests
 {
     public sealed class TeeInputTest
     {
+        [Fact]
+        public void CopiesFromUrlToFile()
+        {
+            var directoryPath = Path.GetTempPath() + "Yaapii.Atoms.Tests.Temp-Files";
```

Comment:
Use our help class Tempdirectory - it is self cleaning when put into a using block.